### PR TITLE
PMM-9306 update victoria metrics version from 1.60.0 to 1.70.0

### PR DESCRIPTION
**Pull request by anton-bystrov**
PMM-9306 update victoria metrics to 1.70.0 version.

### CODE REVIEW

File: rhel/SPECS/victoriametrics.spec [hunk 1]
```diff
@@ -50,6 +50,9 @@ install -D -p -m 0755 ./bin/vmalert-pure %{buildroot}%{_sbindir}/vmalert
 
 
 %changelog
+* Thu Dec  13 2021 Vadim Yalovets <anton.bystrov@simbirsoft.com> - 1.69.0-1
```
Comment (denisok) [inline rhel/SPECS/victoriametrics.spec:53]: Vadim Yalovets? :)

**Review by denisok — APPROVED** [review]

File: rhel/SPECS/victoriametrics.spec (hunk 1 above)
Comment (anton-bystrov) [reply rhel/SPECS/victoriametrics.spec]: Sorry, fixed it :)

File: rhel/SPECS/victoriametrics.spec [hunk 2]
```diff
@@ -50,6 +50,9 @@ install -D -p -m 0755 ./bin/vmalert-pure %{buildroot}%{_sbindir}/vmalert
 
 
 %changelog
+* Thu Dec  13 2021 Anton Bystrov <anton.bystrov@simbirsoft.com> - 1.69.0-1
```
Comment (nikita-b) [inline rhel/SPECS/victoriametrics.spec:53]: Two spaces after Dec :)

**Review by nikita-b — APPROVED** [review]

File: rhel/SPECS/victoriametrics.spec (hunk 2 above)
Comment (anton-bystrov) [reply rhel/SPECS/victoriametrics.spec]: Fixed.

**Review by atymchuk — APPROVED** [review]

**Review by denisok — APPROVED** [review]

**Pull request closed without merging**